webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.scss

```diff
@@ -30,43 +31,15 @@
     }
 
     td {
-      &:hover {
-        cursor: pointer;
-        border: 1px solid #d1d1d1 !important;
-        background-color: #fff !important;
-
-        &:first-child {
-          cursor: auto;
-        }
-      }
-
       &.active {
         font-weight: bold;
       }
     }
-
-    .error-row {
-      .total-col {
-        background-color: #fcc5ca;
-      }
-      .last-col {
-        background-color: #f78c96;
-      }
-    }
-    .warning-row {
-      .total-col {
-        background-color: #fee4cc;
-      }
-      .last-col {
-        background-color: #fdc899;
-      }
-    }
-    .success-row {
-      .total-col {
-        background-color: #d1f0d6;
-      }
-      .last-col {
-        background-color: #a4e0ac;
+    .data-col {
+      &:hover {
+        cursor: pointer;
+        border: 1px solid #d1d1d1 !important;
+        background-color: #def3fe !important;
```

Comment:
- Could you add small margin to the Icons
- can you set the cursor to default on text ? 

![tasks-stopped](https://user-images.githubusercontent.com/26363699/55712594-b29df000-59f7-11e9-89bc-60b61c574a10.png)